### PR TITLE
Added timesformer configuration

### PR DESCRIPTION
**Pull request by AdiaWu**
Co-authored-by: JuheonChu <chuj@dickinson.edu>

# What does this PR do?

Fixes # 19487 (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@ydshieh 
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: utils/documentation_tests.txt [hunk 1]
```diff
@@ -169,8 +169,6 @@ src/transformers/models/speech_to_text/configuration_speech_to_text.py
 src/transformers/models/speech_to_text/modeling_speech_to_text.py
 src/transformers/models/speech_to_text_2/configuration_speech_to_text_2.py
 src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py
-src/transformers/models/speecht5/modeling_speecht5.py
-src/transformers/models/speecht5/tokenization_speecht5.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:173]: These 2 lines should not be deleted.

**Comment by ydshieh** [comment]
Hi @AdiaWu and @JuheonChu, thank you for the contribution 🚀 The doctest is confirmed to pass with this change, but there are 2 lines that should not be deleted in this PR. Once that part is reverted, we are ready to merge 💯

**Comment by AdiaWu** [comment]
Hi @ydshieh Thank you very much for your advice, we will work on it now!

**Comment by ydshieh** [comment]
Hi, @AdiaWu, I am not sure if something changed since yesterday, but currently, the file `documentation_tests.txt` is completely  modified, in particular in [this commit](https://github.com/AdiaWu/transformers/commit/657de28e023ff305fde3356e187b96c29bca1f07)

Why that file is re-created in that commit? Before we can merge this PR, you will have to make that file clean - there should be only 1 line change instead of all file being changed.

**Comment by AdiaWu** [comment]
I am sorry for the trouble. However, I only added line 172 and line 173 as you instructed yesterday. May I ask you what is the one line that you want me to revise? We will try to work on it again.

**Comment by AdiaWu** [comment]
Dear @ydshieh 
and from this site: https://github.com/huggingface/transformers/pull/21446/commits/82d5a6216e3c4596f2213dbee440f12bdaa35fcb. You can check that I only added two lines to the document... Not sure if somewhere went wrong.. 
Agian, sorry for the trouble.

**Comment by ydshieh** [comment]
One commit before, there is unusual changes:

https://github.com/huggingface/transformers/pull/21446/commits/657de28e023ff305fde3356e187b96c29bca1f07

**Comment by AdiaWu** [comment]
@ydshieh Hello, I just wanted to double check. So, what @JuheonChu and I should do is deleting "`src/transformers/utils/documentation_tests.txt`" file. Are we understanding correctly?

**Comment by ydshieh** [comment]
> @ydshieh Hello, I just wanted to double check. So, what @JuheonChu and I should do is deleting "`src/transformers/utils/documentation_tests.txt`" file. Are we understanding correctly?

Yes

**Comment by ydshieh** [comment]
Hi @AdiaWu If you check on the changed file page

https://github.com/huggingface/transformers/pull/21446/files

It shows the file `utils/documentation_tests.txt` is not there (or points to `src/transformers/utils/documentation_tests.txt`.

I suggest you verify the file locally, make sure `utils/documentation_tests.txt` exist, not a symbolic link to other files, and with the expected one-line change you want to add for this PR 🙏.

**Comment by JuheonChu** [comment]
@ydshieh Is the file `utils/documentation_tests.txt` already in our commits?

File: src/transformers/utils/documentation_tests.txt [hunk 2]
```diff
@@ -220,4 +220,5 @@ src/transformers/models/yolos/configuration_yolos.py
 src/transformers/models/yolos/modeling_yolos.py
 src/transformers/models/x_clip/modeling_x_clip.py
 src/transformers/models/yoso/configuration_yoso.py
+src/transformers/models/timesformer/configuration_timesformer.py
```
Comment (JuheonChu) [inline src/transformers/utils/documentation_tests.txt:223]: Here is our commit in the `utils/documentations_test.txt` file to address the timesformer_configuration

**Comment by AdiaWu** [comment]
Dear @ydshieh , 
Since the file "utils/documentation_tests.txt" is missing. @JuheonChu and I just updated the "utils/documentation_tests.txt". There should be no problem with the file now. Please check it whenever you are available and see if there are still some errors this time.

**Comment by ydshieh** [comment]
@AdiaWu 

- The file `src/transformers/utils/documentation_tests.txt` should not be added.
- The file `utils/documentation_tests.txt` should not be deleted.

It seems at some point of your commits, you have done something to these  2 files.

What needs to be done is:

- remove the newly added file `src/transformers/utils/documentation_tests.txt`
- make sure  `utils/documentation_tests.txt` exist, and is not a symbolic link to any other file
- make sure `utils/documentation_tests.txt` is updated with one line `src/transformers/models/timesformer/configuration_timesformer.py`

I hope this is clear and we can merge the PR once everything is fine, thank you.

File: src/transformers/utils/documentation_tests.txt (hunk 2 above)
Comment (ydshieh) [reply src/transformers/utils/documentation_tests.txt:223]: This is not the file `utils/documentations_test.txt`, this is `src/transformers/utils/documentation_tests.txt`, which is newly added in this PR, but it should not be added.

**Comment by AdiaWu** [comment]
Dear @ydshieh , I am sorry for misunderstanding your instructions, we have now deleted the "src/transformers/utils/documentation_tests.txt" file, and the "utils/documentation_tests.txt" is currently in the path with only one line added. I hope this time this PR should be all good and ready to be merged. Thank you very much!

**Comment by JuheonChu** [comment]
> Dear @ydshieh , I am sorry for misunderstanding your instructions, we have now deleted the "src/transformers/utils/documentation_tests.txt" file, and the "utils/documentation_tests.txt" is currently in the path with only one line added. I hope this time this PR should be all good and ready to be merged. Thank you very much!

@ydshieh Thank you for your guidance, and do you mind if you can verify it?

**Comment by ydshieh** [comment]
No worry🤗, and yes it is good now🚀 Thank you so much for making it and the contribution ❤️

**Comment by JuheonChu** [comment]
> No worry🤗, and yes it is good now🚀 Thank you so much for making it and the contribution ❤️

Thank you for your patience!

**Review by ydshieh — APPROVED** [review]

**Pull request merged**